ansys/mapdl/core/logging.py

```diff
@@ -0,0 +1,499 @@
+        The default is ``logging.DEBUG``.
+    to_filet : bool, optional
+        Write log messages to a file. The default is ``False``.
+    to_stdout : bool, optional
+        Write log messages into the standard output (terminal). The default is ``True``.
+    filename : str, optional
+        Name of the file where log messages are written to.
+        The default is ``None``.
+    """
+
+    _file_handler = None
+    _std_out_handler = None
+    _level = logging.DEBUG
+    _instances = {}
+
+    def __init__(self, level=logging.DEBUG, to_file=False, to_stdout=True, filename=FILE_NAME):
+        """
+        Customized logger class for PyAnsys.
+
+        Parameters
+        ----------
+        level : str, optional
+            Level of logging as defined in the package ``logging``. By default _level.
+        to_file : bool, optional
+            To record the logs in a file, by default False
+        to_stdout : bool, optional
+            To output the logs to the standard output, which is the command line. By default True.
+        filename : str, optional
+            Name of the output file. By default FILE_NAME.
+        """
+
+        self._global = logging.getLogger('_Global_')  # Creating default main logger.
+        self._global.setLevel(level)
+        self._global.propagate = True
+        self.level = self._global.level # TODO: TO REMOVE
+
+        # Writing logging methods.
+        self.debug    = self._global.debug
+        self.info     = self._global.info
+        self.warning  = self._global.warning
+        self.error    = self._global.error
+        self.critical = self._global.critical
+        self.log      = self._global.log
+
+        if to_file or filename != FILE_NAME:
+            # We record to file
+            self.log_to_file(filename=filename, level=level)
+
+        if to_stdout:
+            self.log_to_stdout(level=level)
+
+        self.add_handling_uncaught_expections(self._global) # Using logger to record unhandled exceptions
+
+    def log_to_file(self, filename=FILE_NAME, level=LOG_LEVEL):
+        """
+        Add file handler to logger.
+
+        Parameters
+        ----------
+        filename : str, optional
+            Name of the file where the logs are recorded. By default FILE_NAME
+        level : str, optional
+            Level of logging. E.x. 'DEBUG'. By default LOG_LEVEL
+        """
+
+        self = add_file_handler(self, filename=filename, level=level, write_headers=True)
+
+    def log_to_stdout(self, level=LOG_LEVEL):
+        """
+        Add standard output handler to the logger.
+
+        Parameters
+        ----------
+        level : str, optional
+            Level of logging record. By default LOG_LEVEL
+        """
+
+        self = add_stdout_handler(self, level=level)
+
+    ## Copy functions
+    add_stdout_handler = log_to_stdout
+    add_file_handler = log_to_file
+
+    def _make_child_logger(self, sufix, level):
```

Comment:
Using ``getChild`` does not allow to change the logging level, hence the copying the handlers and setting the new log level. 

Having different logging level is interesting in many ways, but fundamentally it is needed for the keyword "loglevel" used in ``launch_mapdl()``. Otherwise, the instance logger inheritate the loglevel from the ``_Global_`` which woudl require for us to change that log level which means edit the library ``__init__`` file or at least, import it as:
```python
from ansys.mapdl.core import LOG
LOG.setLevel(newlevel)
#Probably change the handlers loglevel too
```